e2e-tests/spring-rest-h2-v1/src/test/java/org/evomaster/e2etests/spring/examples/db/preparedstatement/PreparedStatementTest.java

```diff
@@ -0,0 +1,44 @@
+package org.evomaster.e2etests.spring.examples.db.preparedstatement;
+
+import com.foo.rest.examples.spring.db.preparedstatement.PreparedStatementController;
+import io.restassured.http.ContentType;
+import org.evomaster.e2etests.spring.examples.SpringTestBase;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+
+public class PreparedStatementTest extends SpringTestBase {
```

Comment:
create as a well a `PreparedStatementEMTest` in which you run EvoMaster

Reply:
@onurd86 also, the same kind of test needs to be under `spring-rest-openapi-v2`, `spring-rest-mysql` and `spring-rest-postgres` (as the tested code is database dependent)

Reply:
Hi Andrea,

Thanks for your feedback. I have added e2e tests based on our discussion. For writing tests for spring-rest-openapi-v2, spring-rest-mysql, spring-rest-postgres, I am putting this on hold for now according to what I understood from our discussion.